scripts/survival_migration/survivalDataMigration.py

```diff
@@ -0,0 +1,191 @@
+import csv
+import pandas as pd
+import numpy as np
+import os
+import sys
+import argparse
+import shutil
+
+PARENT_PARSER_NAME = 'survival data migration tool'
+NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"]
+MAPPING_FILE_PATH = os.getcwd() + '/survivalStatusVocabularies.txt'
```

Comment:
I don't know why I didn't catch this before, but we shouldn't hard code this filename and accept an optional file - just take one file from the command line - a user of this tool can choose to add to it or not.